src/FirmwareAnalysis/FirmwareAnalysis/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.2.0
```

Comment:
Please move the release notes to under the Upcoming Release section instead of adding a new version section.